lib/tasks/data_hygiene/inconsistent_redirect_finder.rb

```diff
@@ -0,0 +1,16 @@
+class InconsistentRedirectFinder
+  attr_reader :content_items
+
+  def initialize(content_items)
+    @content_items = content_items
+  end
+
+  def items_with_inconsistent_redirects
+    content_items.select do |content_item|
+      next if content_item.format == "redirect"
+
+      route = Rails.application.router_api.get_route(content_item.base_path)
+      route && route.handler == "redirect" && !route.redirect_to.to_s.include?('#')
```

Comment:
It may just be me, but I'm having trouble understanding this. In particular, what's the check with the '#' trying to do?